content/post/70-js-ipfs-0-38.md

```diff
@@ -0,0 +1,119 @@
+---
+date: 2019-08-30
+url: 70-js-ipfs-0-38
+title: js-ipfs 0.38.0 released
+author: Alex Potsides
+---
+
+> Garbage collection, gossipsub, IPNS support for ipfs resolve, oh my!
+
+```console
+$ npm install -g ipfs
+```
+
+# 🔦 Highlights
```

Comment:
Suggest  adding one sentence here with the three main highlights (GC, gossipsub, and resolve). Maybe also worth calling out this is the first release through the new process and we need more early testers??

Reply:
Done!